languages/gutenberg.pot

```diff
@@ -7,23 +7,23 @@ msgstr ""
 msgid "Freeform"
 msgstr ""
 
-#: editor/header/mode-switcher/index.js:25
+#: editor/header/mode-switcher/index.js:30
```

Comment:
It seems important to keep these line numbers up to date, but I worry the changes will be noisy in pull requests. There's also a chance that the developer doesn't run `npm run build` or `npm run dev` after making changes.

It could be neat to set up a bot which automatically commits any necessary `.pot` updates directly to master after a merge, though it's uncertain how this could play out long-term with core integration.